worker/categorize.py

```diff
@@ -395,7 +395,7 @@ def _is_viable_candidate(self, phrase):
         words = phrase.split()
 
         # Ignore stop words
-        if words[0].lower() in STOP_WORDS:
+        if len(words) == 0 or words[0].lower() in STOP_WORDS:
```

Comment:
I don't know. It failed after running for 30 mins and we don't have logging before this point to see what the ```phrase``` being split actually is.